caiman/source_extraction/cnmf/online_cnmf.py

```diff
@@ -359,11 +360,26 @@ def _prepare_object(self, Yr, T, new_dims=None, idx_components=None):
             loaded_model = None
             self.params.set('online', {'sniper_mode': False})
         else:
-            logger.info('Using Torch')
-            path = self.params.get('online', 'path_to_model').split(".")[:-1]
-            model_path = '.'.join(path + ['pt'])
-            loaded_model = PyTorchCNN()
-            loaded_model.load_state_dict(torch.load(model_path))
+            os.environ["KERAS_BACKEND"] = "torch"
+            try:
+                import keras_core as keras
+            except ImportError:
+                import keras
```

Comment:
It'd be good to move this to being a static import; dynamic imports are hard to reason about.